### PR TITLE
[MarkScan] Add the accessible controller help flow

### PR DESCRIPTION
**Pull request by kofi-q**
## Overview

Resolves https://github.com/votingworks/vxsuite/issues/4623

Adding an accessible controller sandbox screen that allows the user to try out the various buttons without triggering the default behaviour - instead, they'll receive audio feedback on what each button does.

The sandbox will be triggered by the "help" button (currently mapped to the old "replay" keybinding, `Shift+R` -- we can clean that up later) and can be exited with another two successive presses on the "help" button (the first press describes the button and tells the voter they can press it again to exit).

## Demo Video or Screenshot [ 🔊 ]

A little long, so recorded with increased speech rate to speed things up a bit:

https://github.com/votingworks/vxsuite/assets/264902/cc3ffc23-c8a5-4d7d-8bad-2cf38cdf0395

## Testing Plan
- New unit tests for the sandbox component and trigger hook
- Broke out a `VoterFlow` component from `app_root` to make testing easier and make it easier to scope the trigger to just voter sessions 

## Checklist

- ~[ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.~

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

**Comment by kshen0** [comment]
Nice! Button descriptions sound good. Should the pause button also say it can be pressed again to resume playback? There's no `Play` button so the behavior might be slightly unintuitive.

File: libs/ui/src/accessible_controllers/accessible_controller_sandbox.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      const key = event.key as T;
+      const audioStringKey = feedbackStringKeys[key];
+      if (!audioStringKey) {
+        return;
+      }
+
+      // Waiting to update the audio string key in the next tick allows the
+      // previous state to get cleared and for the screen reader to stop before
+      // triggering new audio. This enables repeated audio feedback on the
+      // repeated pressed of the same button.
```
Comment (kshen0) [inline libs/ui/src/accessible_controllers/accessible_controller_sandbox.tsx:62]: Typo? Did you mean something like
> This enables repeated audio feedback on the repeated pressing of the same button

File: libs/ui/src/accessible_controllers/accessible_controller_sandbox.test.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+  return {
+    expectLastMockScreenReaderContent: (textContent: string) =>
+      expect(lastReadElement).toHaveTextContent(textContent),
+    renderWithMockScreenReader,
+  };
+}
+
+function simulateKeyPress(key: string) {
+  userEvent.keyboard(key.length === 1 ? key : `[${key}]`);
```
Comment (kshen0) [inline libs/ui/src/accessible_controllers/accessible_controller_sandbox.test.tsx:59]: nit: factor out for DRYness? Similar implementation to https://github.com/votingworks/vxsuite/pull/4740/files#diff-4d3153775fb1fec32f94bd89603d0bcf0828ed2d79f72cefcc61d807dc23b5d6R26

**Review by kshen0 — APPROVED** [review]

File: libs/ui/src/accessible_controllers/accessible_controller_sandbox.tsx (hunk 1 above)
Comment (kofi-q) [reply libs/ui/src/accessible_controllers/accessible_controller_sandbox.tsx:62]: Thanks for catching! Yeah, meant to write `repeated audio feedback on repeated presses`

File: libs/ui/src/accessible_controllers/accessible_controller_sandbox.test.tsx (hunk 2 above)
Comment (kofi-q) [reply libs/ui/src/accessible_controllers/accessible_controller_sandbox.test.tsx:59]: +1 - will do

**Review by kofi-q — COMMENTED** [review]
> Should the pause button also say it can be pressed again to resume playback?

Ah, good catch - I accidentally cut that audio short in the video. The full audio text mentions the unpause functionality: https://github.com/votingworks/vxsuite/blob/main/libs/ui/src/ui_strings/app_strings.tsx#L244

**Pull request merged**